src/SortableList.js

```diff
@@ -154,16 +155,27 @@ export default class SortableList extends Component {
     const {contentHeight, contentWidth, scrollEnabled} = this.state;
     const containerStyle = StyleSheet.flatten([this.props.style, this.state.style]);
     const innerContainerStyle = [styles.container];
+    let refreshControl = this.props.refreshControl;
 
     if (horizontal) {
       innerContainerStyle.push({width: contentWidth});
     } else {
       innerContainerStyle.push({height: contentHeight});
     }
 
+    if(refreshControl) {
+      refreshControl=(
+        <RefreshControl
```

Comment:
I'm not quite sure I understand, but no, we can't pass any element as refreshControl to the ScrollView. It needs to be a `<RefreshControl />` that follows the same React-Native RefreshControl usage:

```
<SortableList
  refreshControl={
    <RefreshControl
      refreshing={this.state.refreshing}
      onRefresh={this._onRefresh.bind(this)}
    />
  } 
/>
```

I want the usage to be transparent from this library's users' point of view. They will just use it as any other RefreshControl in RN. And in the ScrollView inside the sortable list we take care of the RefreshControl on Android by adding the `enabled={scrollEnabled}`.

Should I make this clearer in the readme?